### PR TITLE
iam: add service account enable/disable snippets

### PR DESCRIPTION
**Pull request by mdejong3**
Adding service account enable/disable snippets to go along with this doc addition: https://critique.corp.google.com/#review/267282634

### CODE REVIEW

File: iam/snippets/service_accounts_disable.go [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+	"context"
+	"fmt"
+	"io"
+
+	"golang.org/x/oauth2/google"
+	iam "google.golang.org/api/iam/v1"
+)
+
+// disableServiceAccount disable a service account.
+func disableServiceAccount(w io.Writer, email string) error {
```
Comment (tbpg) [inline iam/snippets/service_accounts_disable.go]: Please add an example `email`.
```suggestion
func disableServiceAccount(w io.Writer, email string) error {
	// email := "..."
```

https://github.com/GoogleCloudPlatform/golang-samples/blob/master/CONTRIBUTING.md#function-arguments

File: iam/snippets/service_accounts_enable.go [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+	"fmt"
+	"io"
+
+	"golang.org/x/oauth2/google"
+	iam "google.golang.org/api/iam/v1"
+)
+
+// enableServiceAccount enables a service account.
+func enableServiceAccount(w io.Writer, email string) error {
+	client, err := google.DefaultClient(context.Background(), iam.CloudPlatformScope)
```
Comment (tbpg) [inline iam/snippets/service_accounts_enable.go]: Is there a way to do this with the cloud.google.com/go/iam package? That's generally preferred over the google.golang.org/api/iam packages. I don't see an obvious way to do it at first glance.

File: iam/snippets/service_accounts_enable.go [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+	if err != nil {
+		return fmt.Errorf("google.DefaultClient: %v", err)
+	}
+	service, err := iam.New(client)
+	if err != nil {
+		return fmt.Errorf("iam.New: %v", err)
+	}
+
+	request := new(iam.EnableServiceAccountRequest)
```
Comment (tbpg) [inline iam/snippets/service_accounts_enable.go]:
```suggestion
	req := &iam.EnableServiceAccountRequest{}
```

Does the same thing, but follows the pattern of other samples.

File: iam/snippets/service_accounts_enable.go (hunk 2 above)
Comment (mdejong3) [reply iam/snippets/service_accounts_enable.go]: I don't believe so--it has a completely different set of functions that doesn't include the New(client) function or Enable(ServiceAccount)/Disable(ServiceAccount).

File: iam/snippets/service_accounts_enable.go (hunk 3 above)
Comment (mdejong3) [reply iam/snippets/service_accounts_enable.go]: Done

File: iam/snippets/service_accounts_disable.go (hunk 1 above)
Comment (mdejong3) [reply iam/snippets/service_accounts_disable.go]: Done

File: iam/snippets/service_accounts_disable.go [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+import (
+	"context"
+	"fmt"
+	"io"
+
+	"golang.org/x/oauth2/google"
+	iam "google.golang.org/api/iam/v1"
+)
+
+// disableServiceAccount disable a service account.
```
[the remark this answers is not in the document]
Comment (mdejong3) [reply iam/snippets/service_accounts_disable.go]: Done

**Pull request merged**